package.json

```diff
@@ -2,16 +2,35 @@
   "name": "wordspaces",
   "version": "0.0.0",
   "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/efishery/wording.git"
+  },
+  "bugs": {
+    "url": "https://github.com/efishery/wording/issues"
+  },
+  "engines": {
+    "node": ">=14.0.0"
+  },
   "workspaces": [
-    "packages/*"
+    "tooling/**",
+    "packages/*",
+    "!tooling/config/*"
```

Comment:
This is like a _flag_, when we write `!<PATH>` it's mean not will be publish/release by `multi-semantic-release`.